lib/std/sysrand.nim

```diff
@@ -144,7 +144,7 @@ elif defined(windows):
     hAlgorithm: BCRYPT_ALG_HANDLE,
     pbBuffer: PUCHAR,
     cbBuffer: ULONG,
-    dwFlags: ULONG
+    dwFlags: ULONG,
```

Comment:
trailing comma?

Reply:
Ye, for consistency with the other lines.

Reply:
this should be discussed and change the nep1 or contributing guide.

Reply:
No trailing commas in parameter lists please.

Reply:
i agree with no commas in parameter lists; but IMO nep1 should recommend trailing commas for enums if there is 1 comma, eg:
```nim
type A1 = enum # good
  k1
  k2
type A2 = enum # good
  k1,
  k2,
type A3 = enum # bad
  k1,
  k2
```